src/platform/user/profile/utilities/index.js

```diff
@@ -108,6 +108,8 @@ export function mapRawUserDataToState(json) {
     userState.status = getErrorStatusDesc(errorStatus);
   } else {
     userState.status = vaProfile.status;
+    userState.isCernerPatient = vaProfile.isCernerPatient;
+    userState.facilities = vaProfile.facilities;
```

Comment:
I will change this code (and the selector below) to match https://github.com/department-of-veterans-affairs/vets-website/pull/11952 when that is merged.